src/MobileUI/Controls/ProfileStats.xaml

```diff
@@ -22,7 +22,12 @@
         Spacing="15"
         Padding="0,15">
         <Grid>
+            <Grid.GestureRecognizers>
+                <TapGestureRecognizer Command="{Binding ChangeProfilePictureCommand}" />
+            </Grid.GestureRecognizers>
+            
             <toolkit:AvatarView ImageSource="{Binding ProfilePic}"
+                                Text=""
```

Comment:
Do we need empty `Text` here?

Reply:
As discussed, this is to prevent "?" showing before an image loads. I've added a comment in the code to better explain this.